samples/HelloAndroid/codebase/exe/exe.sharpmake.cs

```diff
@@ -88,6 +88,7 @@ public ExePackaging() : base(typeof(CommonTarget))
 
             AddTargets(CommonTarget.GetAndroidTargets());
 
+            //plaugin and gradle version are taken from https://gitlab-ncsa.ubisoft.org/Sharpmake/sharpmake/-/merge_requests/230
```

Comment:
typo on plugin, and can't refer to out internal gitlab...

Reply:
Hmm I'm not sure what to put to explain the version numbers, since I'm just following the tutorials from that link

Reply:
Yeah but those came from somewhere, it's not sharpmake that invented them!